cmd/gateway-anonymous.go

```diff
@@ -47,3 +52,19 @@ func anonErrToObjectErr(statusCode int, params ...string) error {
 
 	return errUnexpected
 }
+
```

Comment:
Perhaps we can say why we added this like we added this since default Go sets MaxIdleConns as 2